test/src/test/java/com/distkv/dst/test/benchmark/core/concepts/DstSortedListsImplSimpleBenchmark.java

```diff
@@ -0,0 +1,197 @@
+package com.distkv.dst.test.benchmark.core.concepts;
+
+import com.distkv.dst.common.DstTuple;
+import com.distkv.dst.common.entity.sortedList.SortedListEntity;
+import com.distkv.dst.core.struct.slist.SortedList;
+import com.distkv.dst.core.struct.slist.SortedListLinkedImpl;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class DstSortedListsImplSimpleBenchmark {
+
+  private static final int maxOperationtimes = 10;
+
+  private static final int minValueInPutList = -100000;
+
+  private static final int maxValueInPutList = 100000;
+
+  private static final int lengthInPutList = 1000000;
+
+  public static void main(String[] args) {
```

Comment:
Could you please add the results of this benchmark at your PR descriptor?

Reply:
Ok, I get it.